src/components/ContactPolicy/ContactPolicy.js

```diff
@@ -1,20 +1,22 @@
 import React, { useState } from 'react';
 import { Helmet } from 'react-helmet';
-import { useIntl } from 'react-intl';
+import { FormattedMessage, useIntl } from 'react-intl';
 import HeaderSection from '../shared/HeaderSection/HeaderSection';
 import { Breadcrumb, BreadcrumbItem } from '../shared/Breadcrumb/Breadcrumb';
 import { FormattedMessageMarkdown } from '../../i18n/FormattedMessageMarkdown';
 import { Switch } from '../shared/Switch/Switch';
 import { FieldGroup, FieldItem, NumberField, SubmitButton } from '../form-helpers/form-helpers';
 import { Form, Formik } from 'formik';
 import useTimeout from '../../hooks/useTimeout';
+import { InjectAppServices } from '../../services/pure-di';
 
-export const ContactPolicy = () => {
-  const [active, toggleActive] = useState(false);
-  const [formSubmitted, setFormSubmitted] = useState(false);
+export const ContactPolicy = InjectAppServices(({ dependencies: { experimentalFeatures } }) => {
+  const [isActive, setIsActive] = useState(false);
```

Comment:
How about this? We talked with the team and most of us are in favor of not using is.
```suggestion
  const [active, setActive] = useState(false);
```

Reply:
OK. Is this for all fields or just for 'active' field?

Reply:
Creo que este comentario puede responder tu pregunta: https://github.com/FromDoppler/doppler-webapp/pull/1461#discussion_r663217722